<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mnemo</class>
 <widget class="QLabel" name="mnemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>935</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1215</width>
    <height>935</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>mnemo</string>
  </property>
  <property name="pixmap">
   <pixmap resource="mnemo/mnemo.qrc">:/mnemo/mnemo.svg</pixmap>
  </property>
  <property name="alignment">
   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
  </property>
  <widget class="QFrame" name="ap_1">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>100</y>
     <width>350</width>
     <height>421</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>421</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="ap_2">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>100</y>
     <width>350</width>
     <height>421</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>421</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="ap_3">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>100</y>
     <width>350</width>
     <height>421</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>421</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="c2_h_1">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>584</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c1_h_1">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>576</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c2_h_2">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>585</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c1_h_2">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>577</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c2_h_3">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>584</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/pict/pict/lib/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/pict/pict/lib/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="c1_h_3">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>576</y>
     <width>20</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 20px;
min-height: 32px;
max-width: 20px;
max-height: 32px;
}
QCheckBox:checked
{
	
	
	image: url(:/pict/pict/lib/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/pict/pict/lib/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>660</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>660</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>660</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bnCall_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>97</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Управління</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bnCall_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>97</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Управління</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bnCall_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>97</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Управління</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_State_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_State_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_State_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="pn_L_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>553</y>
     <width>21</width>
     <height>84</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pn_L_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>553</y>
     <width>21</width>
     <height>84</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pn_L_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>552</y>
     <width>21</width>
     <height>84</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_T_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>150</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_T_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>150</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_T_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>150</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mnemo/mnemo.qrc"/>
 </resources>
 <connections/>
</ui>
